### ISSUE TITLE
How to set parameters for optimal results?

### ISSUE THREAD

**Issue by Sheagu** [catezi/MAPT#5]
Hello, catezi! I'm reproducing your reward learning work, and encountered a small issue. I have run `train_football_3_vs_1_reward.sh`, and separately set
Case 1:  `--multi_transformer.use_lstm=false` and `config.use_weighted_sum = False` in MultiPrefTransformer.py
Case 2: `config.pref_attn_embd_dim = 64, config.use_weighted_sum = True` in MultiPrefTransformer.py to avoid dimension inconsistency when passing the `out` tensor to time_weight_head in encoder_decoder_divide.py
Then I plotted the loss curve for both cases. The figure shows that the result of case 1 is significantly better than case 2 with regard to validation loss. However, case 2 is expected to perform better since the corresponding reward model produces better RL training results in the paper (Table 1). 
I suspect that the performance of case 2 may be undermined by suboptimal parameter settings. Could you provide guidance on the optimal parameter settings to help us achieve the expected results? Thank you in advance!
